<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Pixel" failures="1" tests="5" name="com.runnerclass.MenJacketRunnerClass" time="23.224" errors="0" timestamp="2025-02-25T11:54:23 CET" skipped="0">
  <testcase classname="com.runnerclass.MenJacketRunnerClass" name="terminateBrowser" time="2.469">
    <failure message="ERROR OCCURED WHILE CLICKING ELEMENT" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: ERROR OCCURED WHILE CLICKING ELEMENT
at org.junit.Assert.fail(Assert.java:89)
at com.reusablemethods.BaseClass_TestNG.clickOnElement(BaseClass_TestNG.java:154)
at com.pageobjectmodel.LogOutPageObjectModel.getValidLogoutPage(LogOutPageObjectModel.java:27)
at com.runnerclass.MenJacketRunnerClass.terminateBrowser(MenJacketRunnerClass.java:30)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.afterRun(TestRunner.java:877)
at org.testng.TestRunner.run(TestRunner.java:599)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- terminateBrowser -->
  <system-out/>
  <testcase classname="com.runnerclass.MenJacketRunnerClass" name="loginPage" time="5.061"/>
  <system-out/>
  <testcase classname="com.runnerclass.MenJacketRunnerClass" name="searchProductPage" time="5.145"/>
  <system-out/>
  <testcase classname="com.runnerclass.MenJacketRunnerClass" name="shippingPage" time="3.453"/>
  <system-out/>
  <testcase classname="com.runnerclass.MenJacketRunnerClass" name="checkoutPage" time="7.096"/>
  <system-out/>
</testsuite> <!-- com.runnerclass.MenJacketRunnerClass -->
